azure-communication-ui/azure-communication-ui-demo-app/src/main/java/com/azure/android/communication/ui/callingcompositedemoapp/launcher/CallingCompositeJavaLauncher.java

```diff
@@ -57,15 +57,15 @@ public void launch(final Context context,
 
         if (groupId != null) {
             final GroupCallOptions groupCallOptions =
-                    new GroupCallOptions(context, communicationTokenCredential, groupId, displayName);
```

Comment:
Since the normal use case is to create Options right before launch, the context really only needs to be moved down two lines (from construction of options, to the launch call)